html/semantics/scripting-1/the-script-element/module/import-meta/import-meta-url.html

```diff
@@ -20,4 +20,26 @@
   assert_equals(importMetaOnDependentModule.url,
                 base + "/import-meta-dependent.js");
 }, "import.meta.url in a dependent external script");
+
+import { importMetaOnRootModule as hashedImportMetaOnRootModule1,
+         importMetaOnDependentModule as hashedImportMetaOnDependentModule1 }
+       from "./import-meta-root.js#1";
+
+import { importMetaOnRootModule as hashedImportMetaOnRootModule2,
+         importMetaOnDependentModule as hashedImportMetaOnDependentModule2 }
+       from "./import-meta-root.js#2";
+
+test(() => {
+  assert_equals(hashedImportMetaOnRootModule1.url,
+                base + "/import-meta-root.js#1");
+  assert_equals(hashedImportMetaOnRootModule2.url,
+                base + "/import-meta-root.js#2");
+
+  // Must not be affected
+  assert_equals(hashedImportMetaOnDependentModule1.url,
+                base + "/import-meta-dependent.js");
+  assert_equals(hashedImportMetaOnDependentModule2.url,
+                base + "/import-meta-dependent.js");
+}, "import.meta.url when importing the module with different fragments");
```

Comment:
Would you mind creating a new test html instead of adding this to existing test?
It would be easier for all browsers to track their progress on interoperability, and scoped tests makes it easier to debug.

Reply:
From Blink implementer point-of-view, it is easier to have fine-grained tests to find partial regressions.
Assuming that the new tests currently wouldn't pass on Blink, it would be easier for us to track new test failing, rather than old test suddenly start failing. We need to bisect if it is caused by a regression from other CLs or it was failing from the beginning in the latter case.